src/avro/resources/dictionary.py

```diff
@@ -830,9 +830,62 @@
         # Remapped words.
         "exceptions": {
             "বাংলাদেশ": "Bangladesh",
+            "ভারত": "Bharot",
+            "পাকিস্তান": "Pakistan",
+            "শ্রীলঙ্কা": "Srilanka",
+            "নেপাল": "Nepal",
+            "ভুটান": "Bhutan",
+            "মালদ্বীপ": "Maldives",
+            "মালয়েশিয়া": "Malaysia",
+            "সিঙ্গাপুর": "Singapore",
+            "চীন": "Chin",
+            "জাপান": "Japan",
+            "কোরিয়া": "Korea",
+            "ইন্দোনেশিয়া": "Indonesia",
+            "থাইল্যান্ড": "Thailand",
+            "ফিলিপাইন": "Philippines",
+            "ভিয়েতনাম": "Vietnam",
+            "সৌদি": "Saudi",
+            "আরব": "Arab",
+            "কাতার": "Qatar",
+            "ওমান": "Oman",
+            "বাহরাইন": "Bahrain",
```

Comment:
All is great, now remove the English words which don't exist at all in the vocabulary.